dcos/package.py

```diff
@@ -58,10 +58,13 @@ def install_app(pkg, revision, init_client, options, app_id):
     init_desc = pkg.marathon_json(revision, options)
 
     if app_id is not None:
-        logger.debug('Setting app ID to "%s" (was "%s")',
-                     app_id,
-                     init_desc['id'])
-        init_desc['id'] = app_id
+        if not isinstance(init_desc, list):
+            logger.debug('Setting app ID to "%s" (was "%s")',
+                         app_id,
+                         init_desc['id'])
+            init_desc['id'] = app_id
+        else:
+            logger.debug('Not setting app ID for multi app json')
```

Comment:
The app_id has to be unique in Marathon. When I have a marathon.json with multiple apps I can't set the app_id for all of them to the same value. I could set it for the first app found in the list, but that feels like odd behaviour. We could extend it so that app_id could also be a list and then override them in the order they were supplied, but again that seems like a very artificially constructed use case.